realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -2658,7 +2658,8 @@ public void isEmpty_illegalFieldTypeThrows() {
                     case STRING_TO_DECIMAL128_MAP:
                     case STRING_TO_LINK_MAP:
                     case STRING_SET:
-                        // TODO: add queries for dictionaries
+                    case INTEGER_SET:
+                        // TODO: what about dictionaries and sets?
```

Comment:
I would like to wait with this at least in the current form. I don't think breaking the pattern of using `AllTypes` here is a good idea - and I would have to because if I add a dictionary or a set field in `AllTypes` I'll be breaking tons of tests due to missing support for `insert`/`createFromJson`.

I created a couple of issues (dictionary: https://jira.mongodb.org/browse/RJAVA-917 and set: https://jira.mongodb.org/browse/RJAVA-944) to handle this when we add support for those methods.